### PR TITLE
fix: removed flux wasm crate and moved it to lsp

### PR DESCRIPTION
**Pull request by BarWeiner1**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

**Comment by nathanielc** [comment]
Are there stale dependencies in the Cargo.toml as a result of this change? Does this mean that the `flux` repo doesn't need any wasm knowledge?

**Comment by BarWeiner1** [comment]
I am pretty sure the flux repo no longer needs any wasm knowledge, which dependencies from Cargo.toml can we safely remove because of that?

File: Makefile [hunk 1]
```diff
@@ -108,7 +108,6 @@ staticcheck:
 		`go list ./... | grep -v '\/flux\/stdlib\>'`
 	GO111MODULE=on ./gotool.sh honnef.co/go/tools/cmd/staticcheck ./stdlib/...
 
-test: test-go test-rust test-flux test-wasm
```
Comment (scbrickley) [inline Makefile:111]: We don't want to remove this line entirely. This is what CI uses to run tests. We just want to remove `test-wasm` from the end of this line.

**Review by scbrickley — CHANGES_REQUESTED** [review]
Just one change, but other than that this looks fine.

**Review by scbrickley — APPROVED** [review]
LGTM

**Review by rockstar — APPROVED** [review]

**Pull request merged**